.github/workflows/stale.yml

```diff
@@ -11,7 +11,7 @@ on:
       days-before-close:
         description: The idle number of days before closing the stale issues or the stale pull requests (due to the stale label).
         type: number
-        default: 7
+        default: 14
```

Comment:
I'm still unsure if it's necessary to set this to 14. I think 7 days is plenty of time to react to the `stale` label on a 60 day old (and inactive) issue.

Reply:
True, I don't have a strong opinion for or against, so we can keep it to 7 days.